.github/pull_request_template.md

```diff
@@ -24,6 +23,15 @@ Thanks for submitting a pull request! Please provide enough information so that
 
 Fixes #<!--Add related issue number here.-->
 
+**Did you add tests for your changes?**
```

Comment:
Your PR has too many unnecessary files that don’t relate to your issue. You can from the PR by running the commands below from the root directory of the repository

git add -u
git reset HEAD path/to/file
git commit

Please apply these changes to this file.